portal-user-datastore-rdbms/src/test/resources/schema-hsqldb-test.sql

```diff
@@ -16,4 +17,7 @@ create table portal_user (
 );
 
 create index idx_portal_user_customerId on portal_user (customerId);
-create unique index portal_user_customerId_username on portal_user (customerId, username);
\ No newline at end of file
+create unique index portal_user_customerId_username on portal_user (customerId, username);
+
+alter table portal_user add column IF NOT EXISTS roles varchar(600);
+update portal_user set roles = '[' + role + ']' where roles is null
```

Comment:
should be :
update portal_user set roles = '[' || role || ']' where roles is null;

Reply:
This was already updated from a previous commit. It should be the same as your comment